membership-attribute-service/app/services/SupporterRatePlanToAttributesMapper.scala

```diff
@@ -61,297 +72,320 @@ object SupporterRatePlanToAttributesMapper {
       GuardianPatronExpiryDate = Some(supporterRatePlanItem.termEndDate),
     )
   val guardianPatronProductRatePlanId = "guardian_patron"
-  val productRatePlanMappings: Map[Stage, Map[List[ProductRatePlanId], AttributeTransformer]] =
+
+  private val prodMappings: Map[List[ProductRatePlanId], AttributeTransformer] = Map(
```

Comment:
I don't see that this refactor is giving us any value, we now just have to map from stage to a named variable below

Reply:
It's just splitting it into smaller, more manageable chunks rather than map of maps declared at once.
There's still one big map.